jobs/influxdb/templates/influxd_ctl.erb

```diff
@@ -78,16 +78,18 @@ daemon=/var/vcap/packages/influxdb/influxd
 bootstrapper=/var/vcap/packages/bootstrapper/bootstrapper
 
 # pid file for the daemon
-pidfile=/var/vcap/store/influxdb/influxd.pid
+pidfile=/var/vcap/sys/run/influxdb/influxd.pid
 
 # Configuration file
 config=/var/vcap/jobs/influxdb/config/influxdb.conf
 
-mkdir -p /var/vcap/store/influxdb
+mkdir -p $(basename $pidfile)
```

Comment:
True, let me fix that

On Thu, Sep 8, 2016, 10:20 AM Alex Suraci <notifications@github.com> wrote:

> In jobs/influxdb/templates/influxd_ctl.erb
> <https://github.com/vito/influxdb-boshrelease/pull/9#discussion_r78015254>
> :
>
> >
> >  # Configuration file
> >  config=/var/vcap/jobs/influxdb/config/influxdb.conf
> >
> > -mkdir -p /var/vcap/store/influxdb
> > +mkdir -p $(basename $pidfile)
>
> This should be dirname right?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/vito/influxdb-boshrelease/pull/9/files/a8ce518651ee9689bf6d644ffaca0b112499b8dd#r78015254>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AASKlWDYVoH43iw_7ydlfxPtQD94dEpQks5qoBmwgaJpZM4J3EOA>
> .
>